kubernetes/ansible/roles/flink-jobs-deploy/defaults/main.yml

```diff
@@ -244,6 +244,7 @@ dialcode_external_indexer_parallelism: 1
 dialcode_metric_indexer_parallelism: 1
 schema_definition_cache_expiry_in_sec: 14400
 search_indexer_ignored_fields: ["responseDeclaration", "body", "options", "lastStatusChangedOn", "SYS_INTERNAL_LAST_UPDATED_ON", "sYS_INTERNAL_LAST_UPDATED_ON"]
+search_indexer_restrict_objectTypes: ["EventSet", "EventSetImage", "Event", "EventImage", "Questionnaire", "Misconception", "FrameworkType", "Concept", "Misconception", "Language", "Reference", "Dimension", "Method", "Library", "Domain", "Api"]
```

Comment:
Keep all variables lowercase in ansible code.

Reply:
Okay, thanks for the response.